recipes/wally/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set version = "0.2.3" %}
+{% set revision = "93442348984044ea12608c323b7480487ab70eba" %}
+
+package:
+  name: wally
+  version: '{{ version }}'
+
+source:
+  git_url: https://github.com/tobiasrausch/wally.git
+  git_rev: {{ revision }}
+  md5: unused
+  patches:
+    - Makefile.patch
+
+build:
+  number: 0
+  skip: True  # [osx]
+
+requirements:
+  build:
+    - make
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - cmake
+    - autoconf
+    - automake
+  host:
+    - libpng
+    - zlib
+    - bzip2
+    - xz
+    - boost-cpp
+    - htslib
+  run:
+    - libpng
+    - zlib
+    - bzip2
+    - xz
+    - boost-cpp
```

Comment:
Does this dynamically link against boost? Otherwise it's not needed here.

Reply:
Links indeed dynamically against boost.

Reply:
There is also a static compile using `make STATIC=1 all` but that failed to build [d30039](https://github.com/bioconda/bioconda-recipes/pull/31716/commits/d300391d3439aa574ecca4aaa7a2007286c9fa33).